network/src/noise/handshake.rs

```diff
@@ -221,6 +227,11 @@ impl NoiseUpgrader {
         TSocket: AsyncRead + AsyncWrite + Unpin,
         F: Fn() -> [u8; AntiReplayTimestamps::TIMESTAMP_SIZE],
     {
+        // increment counter for metrics
+        counters::LIBRA_NETWORK_NOISE_HANDSHAKE
+            .with_label_values(&["initiator", self.auth_mode.to_str()])
+            .inc();
```

Comment:
maybe we should push down the network_id into NoiseUpgrader and include that, then rename the "role_type" label into "connection_origin" x { "inbound", "outbound" }. also rename the "network_type" label to "auth_mode"?

Reply:
if we have the `network_id` we don't need the `auth_mode` right?

Reply:
right, i think we can just use `network_id` instead of this `auth_mode`/`network_type`